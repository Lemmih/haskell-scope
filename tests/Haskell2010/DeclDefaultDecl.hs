module DeclDefault where

data A
data B

default ()
default (A,B)
