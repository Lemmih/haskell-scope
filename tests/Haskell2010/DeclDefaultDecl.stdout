Scope errors:

Definitions:
  Definition 1, from DeclDefault.decl0:
    data [4mA[0m
  Definition 2, from DeclDefault.decl1:
    data [4mB[0m

Use sites:
  Definition used: 1
    default ([4mA[0m,B)
  Definition used: 2
    default (A,[4mB[0m)
